packages/shared/src/components/modals/BasePostModal.tsx

```diff
@@ -49,17 +49,20 @@ function BasePostModal({
 
   const onScroll = useCallback(
     (event: Event) => {
+      if (!post.id) {
+        return;
+      }
       const targetElement = event.target as HTMLElement;
       logEvent({
         event_name: LogEvent.PageScroll,
         target_type: TargetType.Post,
-        target_id: post?.id,
+        target_id: post.id,
         extra: JSON.stringify({
           scrollTop: targetElement.scrollTop,
         }),
       });
     },
-    [logEvent, post?.id],
+    [logEvent, post.id],
```

Comment:
If post is undefined this will break. So I don't think we need to change it as well